data/xml/2021.latechclfl.xml

```diff
@@ -0,0 +1,241 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Same here. Don’t create a new venue. Use `2021.latech-1` as the volume name, then cross-list with `clfl`.

Reply:
Here, it seems the venues have merged. So forget this comment.